getting-started/index.rst

```diff
@@ -99,6 +166,21 @@ directory that was created earlier when you cloned the source code:
 To download tools, bootstrap will use ``wget``, ``curl`` or
 ``lwp-request`` if installed.  If not, it will give an error telling
 you the URL to download from by hand and where to copy the file to.
+If you wish to prevent bootstrap from downloading and building the autotools
+pass the ``--without-autotools`` option.  You can force it to delete the downloaded
```

Comment:
This is out of date and `--without-autotools` is now ignored with a warning.

Now we'll use installed version of autotools by default if they're new enough and we don't have patches to apply (which currently we do for automake and libtool because we needed some small patches for MSVC support and I've failed to get these accepted upstream yet).  The user can now specify `--download-tools=(always|ifneeded|never)` (the default is `ifneeded`).

Reply:
👍